hera_cal/smooth_cal.py

```diff
@@ -182,15 +263,28 @@ def time_freq_2D_filter(gains, wgts, freqs, times, freq_scale=10.0, time_scale=1
                     and fringe rate = 0
             'plus': produce a separable calibration solution by only keeping modes with 0 delay,
                     0 fringe rate, or both
-        window: window function for filtering applied to the filtered axis.
+        window: window function for filtering applied to the filtered axis. Only used with when method used is 'CLEAN'.
             See aipy.dsp.gen_window for options.
         maxiter: Maximum number of iterations for aipy.deconv.clean to converge.
+        method: Algorithm used to smooth calibration solutions. Either 'CLEAN' or 'DPSS':
+            'CLEAN': uses the CLEAN algorithm to
+            'DPSS': uses discrete prolate spheroidal sequences (DPSS) to filter calibration solutions
+        design_matrix: Matrix of DPSS filters calculated using smooth_cal.dpss_filters. If matrix is not provided,
+            one will be calculated using smooth_cal.dpss_filters and the time and frequency scale. Only used
+            when the method used is 'DPSS'
+        sol_matrix: Solution matrix obtained from performing linear least squares with 2D DPSS vectors
+        eigenval_cutoff: sinc_matrix eigenvalue cutoffs to use for included dpss modes.
+            Only used when the filtering method is 'DPSS'
+        skip_flagged_edges : bool, optional
```

Comment:
Do we generally want to use this when DPSS is true? If so, then perhaps this should be default True (but still ignored if CLEAN is True)

Reply:
Yeah, I think in most cases we probably would want to. I set the default to True